webpack.config.ts

```diff
@@ -4,13 +4,17 @@ import { Configuration as WebpackConfiguration } from "webpack";
 import { Configuration as WebpackDevServerConfiguration } from "webpack-dev-server";
 import * as path from "path";
 import { ConsoleRemotePlugin } from "@openshift-console/dynamic-plugin-sdk-webpack";
+const dotenv = require('dotenv')
+const webpack = require('webpack')
```

Comment:
Let's use `import` instead of `require` (e.g. `import { DefinePlugin } from 'webpack';`)

Reply:
fixed